tasks/Functions-billing/timeoutRequest.js

```diff
@@ -0,0 +1,24 @@
+const { networkConfig } = require("../../network-config")
+
+task("functions-timeout-requests", "Times out expired requests")
+  .addParam("requestids", "1 or more request IDs to timeout")
+  .setAction(async (taskArgs) => {
+    if (network.name === "hardhat") {
+      throw Error(
+        'This command cannot be used on a local hardhat chain.  Specify a valid network.'
+      )
+    }
+
+    console.log(
+      `Timing out requests ${taskArgs.requestids} on ${taskArgs.network}`
+    )
+
+    const RegistryFactory = await ethers.getContractFactory("FunctionsBillingRegistry")
+    const registry = await RegistryFactory.attach(networkConfig[network.name]["functionsBillingRegistryProxy"])
+
+    await registry.timeoutRequests(taskArgs.requestids)
+
+    console.log(
+      `Requests ${taskArgs.requestids} are no longer pending or have been timed out`
```

Comment:
A more foolproof way to do this might be to check for an event to be emitted, similar to how the `request` command does.

Looks like timed out requests will `emit RequestTimedOut(requestId);`